### DISCUSSION TITLE
Blank screen on load

### DISCUSSION THREAD

**Discussion by mklein994** [community/community#10647] [Other Feature Feedback, Questions, & Ideas] [answered]
When navigating to https://cs.github.com, I see a blank white screen, with errors in the JavaScript console.

It consistently happens on both Firefox[^firefox] and Google Chrome[^chromium], even when using a private window (incognito). If you need any other troubleshooting information, I can give that as well.

Looking through the stacktrace in devtools, it appears it's because my GitHub account is not part of an organization.

Copied from the debugger, with a comment added showing where it breaks:
```js
// ...
    var Ji = 'GitHub Code Search',
    Zi = [
      'currentUser'
    ];
    var el = function (e) {
      var t = e.currentUser,
      n = Un(e, Zi);
      if (!t) return null;
      var r = t.orgs;
      if (!r.some((function (e) {
      //   ^--- TypeError thrown here: `r` is `null`
        return e.protected
      }))) return null;
      var o = window.location.pathname + window.location.search + window.location.hash;
      return (0, Da.jsx) (tr, Fa(Fa({
      }, n), {
// ...
```

<details>
<summary>Full stacktrace</summary>
<pre>
TypeError: can't access property "some", r is null
    el https://cs.github.com/static/js/main.67ab0798.js:2
    li https://cs.github.com/static/js/main.67ab0798.js:2
    Kl https://cs.github.com/static/js/main.67ab0798.js:2
    _s https://cs.github.com/static/js/main.67ab0798.js:2
    Ts https://cs.github.com/static/js/main.67ab0798.js:2
    Es https://cs.github.com/static/js/main.67ab0798.js:2
    vs https://cs.github.com/static/js/main.67ab0798.js:2
    Yo https://cs.github.com/static/js/main.67ab0798.js:2
    unstable_runWithPriority https://cs.github.com/static/js/main.67ab0798.js:2
    Go https://cs.github.com/static/js/main.67ab0798.js:2
    Yo https://cs.github.com/static/js/main.67ab0798.js:2
    Qo https://cs.github.com/static/js/main.67ab0798.js:2
    Fs https://cs.github.com/static/js/main.67ab0798.js:2
    unstable_runWithPriority https://cs.github.com/static/js/main.67ab0798.js:2
    Go https://cs.github.com/static/js/main.67ab0798.js:2
    Rs https://cs.github.com/static/js/main.67ab0798.js:2
    Is https://cs.github.com/static/js/main.67ab0798.js:2
    O https://cs.github.com/static/js/main.67ab0798.js:2
    onmessage https://cs.github.com/static/js/main.67ab0798.js:2
    95 https://cs.github.com/static/js/main.67ab0798.js:2
    n https://cs.github.com/static/js/main.67ab0798.js:2
    224 https://cs.github.com/static/js/main.67ab0798.js:2
    n https://cs.github.com/static/js/main.67ab0798.js:2
    534 https://cs.github.com/static/js/main.67ab0798.js:2
    n https://cs.github.com/static/js/main.67ab0798.js:2
    168 https://cs.github.com/static/js/main.67ab0798.js:2
    n https://cs.github.com/static/js/main.67ab0798.js:2
    <anonymous> https://cs.github.com/static/js/main.67ab0798.js:2
    <anonymous> https://cs.github.com/static/js/main.67ab0798.js:2
    <anonymous> https://cs.github.com/static/js/main.67ab0798.js:2
main.67ab0798.js:2:157429
</pre>
</details>

[^firefox]: Firefox Developer Edition 97.0b8
[^chromium]: Chromium Version 97.0.4692.99 (Official Build) Arch Linux (64-bit)

**Comment by look** [comment]
Thank you for the detailed report. I was able to replicate this with a test user.

**Comment by KimViet** [comment]
I also got the same bug

**Comment by colinwm** [comment, marked as answer]
Sorry everyone! It's fixed now.
